onnx/test/test_backend_onnxruntime.py

```diff
@@ -155,6 +155,34 @@ def run_node(cls, node, inputs, device=None, outputs_info=None, **kwargs):
         "|test_quantizelinear_axis"  # y_scale must be a scalar or 1D tensor of size 1.
         "|test_quantizelinear"  # No corresponding Numpy type for Tensor Type.
         "|test_affine_grid_"  # new IR version 9 and opset version 20 not supported yet.
+        "|test_quantizelinear_uint4"  # No corresponding Numpy type for Tensor Type.
+        "|test_quantizelinear_int4"  # No corresponding Numpy type for Tensor Type.
+        "|test_dequantizelinear_uint4"  # No corresponding Numpy type for Tensor Type.
+        "|test_dequantizelinear_int4"  # No corresponding Numpy type for Tensor Type.
+        "|test_cast_UINT4_to_FLOAT"  # No corresponding Numpy type for Tensor Type.
+        "|test_cast_INT4_to_FLOAT"  # No corresponding Numpy type for Tensor Type.
+        "|test_cast_UINT4_to_FLOAT16"  # No corresponding Numpy type for Tensor Type.
+        "|test_cast_INT4_to_FLOAT16"  # No corresponding Numpy type for Tensor Type.
+        ")"
+    )
+
+    # Exclude all tests that require IR10 until onnxruntime aligns
```

Comment:
Could you put a `TODO` and link an issue in this repo to better track this? Thanks!

Reply:
Sure thing. Added TODO pointing to #5840